spec/mailers/role_change_mailer_spec.rb

```diff
@@ -10,8 +10,12 @@
     let(:role) { FactoryBot.create(:probation_role, user: delegate) }
     let(:mail) { described_class.notify_role_start(role, user_who_made_the_change) }
 
+    before :each do
+      FactoryBot.create(:board_user_group)
+    end
```

Comment:
You're using this idiom too much. Defining `before :each` makes it so that the code in the `do` block is executed before each invocation of an `it` block. No need to copy it four times.

If necessary, just move the block up one level.

Reply:
Okay, moved it to user_groups_seeds.rb.